### ISSUE TITLE
ModuleNotFoundError: No module named 'maggma'

### ISSUE THREAD

**Issue by janosh** [hackingmaterials/atomate#634]
If I setup `atomate` on new machine (`pip install atomate`) and try

```py
from atomate.vasp.database import VaspCalcDb
```

I get the error

```py
ModuleNotFoundError: No module named 'maggma'
```

After `pip install maggma`, everything if fine. This is weird as I just saw that `maggma>=0.26.0` is included in `install_requires`:

https://github.com/hackingmaterials/atomate/blob/main/setup.py#L24

But this `ModuleNotFoundError: No module named 'maggma'` issue happened on 3 different machines (2 Linux, 1 Mac) so perhaps worth looking into...

**Comment by utf** [comment]
Apologies for this. Maggma has been added to the project requirements but we haven't released a new version since then.

@computron Would it be possible to release a new atomate version?

**Comment by computron** [comment]
I have just released atomate v0.9.9 - closing for now, can reopen if there are still issues afterward.

**Comment by janosh** [comment]
No more issues. 👍
